name/urls.py

```diff
@@ -24,5 +24,5 @@
     path('stats/', views.stats, name='stats'),
     path('<slug:name_id>.json', api.name_json, name='detail-json'),
     path('<slug:name_id>.mads.xml', views.mads_serialize, name='mads-serialize'),
-    path('<slug:name_id>/', views.detail, name='detail')
+    path('<name_id>/', views.detail, name='detail')
```

Comment:
Should it be the same converter for all of these  `name_id` parameters?

Reply:
The other two `name_id`s have ".*",  as `r'(?P<name_id>.*).json$'`, so I think isung `<slug>` is fine there.

Reply:
Since `name_id` refers to the same entity for all of these urls, I think it was an inconsistency how it was before in the regexes to have it in one place as `(?P<name_id>[^/]+)` and another as `(?P<name_id>.*)`. In practice it didn't matter, because the [ids generated by name app for `name_id`](https://github.com/unt-libraries/django-name/blob/master/name/models.py#L197) are only letters and digits meaning regexes `[^/]+` or `.*` or converters `slug` or `str` will all allow the `name_id` to match. If you are strictly translating the regex paths to converters, `.*` matches the [`path` converter](https://docs.djangoproject.com/en/2.2/_modules/django/urls/converters/) which would not work for us here. Based on our `name_id`s being what they are, `slug` or `str` should be sufficient, but I think we should be consistent in URLs on what we use for the same parameter, and since we already have `slug` before this PR, I think we can close this and leave it as-is.